prisma-fmt/src/lint.rs

```diff
@@ -59,3 +59,80 @@ pub(crate) fn run(schema: &str) -> String {
 fn print_diagnostics(diagnostics: Vec<MiniError>) -> String {
     serde_json::to_string(&diagnostics).expect("Failed to render JSON")
 }
+
+#[cfg(test)]
+mod tests {
+    use expect_test::expect;
+    use indoc::indoc;
+
+    fn lint(s: &str) -> String {
+        let result = super::run(s);
+        let value: serde_json::Value = serde_json::from_str(&result).unwrap();
+
+        serde_json::to_string_pretty(&value).unwrap()
+    }
+
+    #[test]
+    fn type_aliases_should_give_a_warning() {
+        let dml = indoc! {r#"
+            datasource db {
+              provider = "postgresql"
+              url      = env("DATABASE_URL")
+            }
+
+            generator client {
+              provider = "prisma-client-js"
+            }
+
+            type MyString = String @default("A")
+
+            model Code {
+              id  String   @id
+              val MyString
+            }
+        "#};
+
+        let expected = expect![[r#"
+            [
+              {
+                "start": 132,
+                "end": 168,
+                "text": "Type aliases are an undocumented feature that is getting deprecated. Please chime in in the issue if you need it: https://github.com/prisma/prisma/issues/9939",
```

Comment:
One "in" to much in the error msg it seems :D

Reply:
I thought so too at first in the previous PR, but I think it's correct: _chime in_ / _in the issue_.